pkg/controller/operators/catalog/operator.go

```diff
@@ -903,6 +904,17 @@ func (o *Operator) syncResolvingNamespace(obj interface{}) error {
 		return err
 	}
 
+	failForwardEnabled := false
+	ogs, err := o.lister.OperatorsV1().OperatorGroupLister().OperatorGroups(namespace).List(labels.Everything())
```

Comment:
suggestion: might be worth putting this in a function just to lean this one out. Not a blocker tho.